.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -0,0 +1,82 @@
+jobs:
+- job: osx_64
+  pool:
+    vmImage: xcode9-macos10.13
```

Comment:
This is a quite recent xcode. I *think* with properly setting CONDA_BUILD_SYSROOT this should be okay (good, even), but updating this version should be considered deliberately.